data/webbrowser.yaml

```diff
@@ -799,9 +799,9 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Configuration']
 ---
-name: ChromeSessions
+name: ChromeSessionsAndTabs
```

Comment:
add `ChromeSessions` as alias for backwards compatibility

Reply:
I removed this when merging with the existing sessions definition